src/message/MentionedUserNotSubscribed.js

```diff
@@ -0,0 +1,71 @@
+/* @flow strict-local */
+
+import React, { Component } from 'react';
+import { View, TouchableOpacity } from 'react-native';
+
+import type { Auth, Stream, Dispatch, Narrow, UserOrBot, GetText } from '../types';
+import { withGetText } from '../boot/TranslationProvider';
+import { connect } from '../react-redux';
+import * as api from '../api';
+import { ZulipButton, RawLabel } from '../common';
+import { getAuth, getStreamInNarrow, getUserForId } from '../selectors';
+import styles from '../styles';
+
+type SelectorProps = {|
+  auth: Auth,
+  stream: { ...Stream },
+  user: UserOrBot,
+|};
+
+type Props = $ReadOnly<{|
+  narrow: Narrow,
+  userId: number,
+  onDismiss: (user: UserOrBot) => void,
+  dispatch: Dispatch,
```

Comment:
nit: blank line to separate the props the caller cares about from the ones supplied by wrappers

Reply:
Fixed.